src/Wallabag/ApiBundle/Controller/EntryRestController.php

```diff
@@ -272,6 +274,13 @@ public function patchEntriesAction(Entry $entry, Request $request)
             $entry->setStarred((bool) $isStarred);
         }
 
+        if (!is_null($progress)) {
```

Comment:
It seems that we don't often use `is_null`, so if we want to keep consistency, we should use `null ===`